.travis.yml

```diff
@@ -19,6 +19,8 @@ os:
   - osx
 script:
   - npm test
-
+after_success:
+  - npm run test:coverage
+  - cat ./coverage/lcov.info | ./node_modules/codecov/bin/codecov
```

Comment:
Ah, I see, forgot that this is for travis specifically. In that case, it's probably fine to leave as-is.